Scripts/WikiUpdater.cs

```diff
@@ -62,93 +132,225 @@ public async Task<bool> Run(CancellationToken cancellationToken)
                 internalName,
-                url,
-                untranslatedSections);
+                pageUrl,
+                untranslatedSections,
+                untranslatedInfobox,
+                noticeSceneName);
             var translatedPage = new Wiki.Page(name,
                 internalName,
-                url,
-                sections);
+                pageUrl,
+                sections,
+                translatedInfobox,
+                noticeSceneName);
 
-            organellePages.Add(new TranslationPair(untranslatedPage, translatedPage));
+            allPages.Add(new TranslationPair(untranslatedPage, translatedPage));
 
-            ColourConsole.WriteSuccessLine($"Populated content for organelle with internal name {internalName}");
+            ColourConsole.WriteSuccessLine($"Populated content for page with internal name {internalName}");
         }
 
-        return organellePages;
+        return allPages;
+    }
+
+    /// <summary>
+    ///   Extracts all the fields of an associated page's infobox and generates translation keys
+    /// </summary>
+    /// <param name="body">The body of the page</param>
+    /// <param name="internalName">Internal name of the page</param>
+    /// <returns>A tuple containing the translated and untranlsated versions of the detected fields</returns>
+    /// <exception cref="InvalidOperationException">Thrown when an infobox is not found</exception>
+    private (List<InfoboxField> Untranslated, List<InfoboxField> Translated)
+        GetInfoBoxFields(IHtmlElement body, string internalName)
+    {
+        ColourConsole.WriteInfoLine($"Extracting infobox content for page with internal name {internalName}");
+
+        // Get the infobox element
+        var infobox = body.QuerySelector(INFO_BOX_SELECTOR) ?? throw new InvalidOperationException(
+            $"Did not find infobox on page with internal name {internalName}");
+
+        var translated = new List<InfoboxField>();
+        var untranslated = new List<InfoboxField>();
+
+        foreach (var row in infobox.Children)
+        {
+            if (row.Children.Length <= 1)
+                continue;
+
+            // Parse the HTML table to get keys and values
+            var value = row.Children.Where(e => e.LocalName == "td").First();
+            var translatedKey = row.Children.Where(e => e.LocalName == "th").First().TextContent.Trim();
+            var id = value.Id;
+
+            if (id == null)
+                continue;
+
+            var textContent = value.TextContent.Trim();
+
+            if (textContent == internalName)
+                continue;
+
+            // Format the found content for use in translation files
+            var untranlsatedKey = id.Replace("#", string.Empty).ToUpperInvariant().Replace("-", "_");
+            var untranslatedValue = textContent.ToUpperInvariant()
+                .Replace(' ', '_')
+                .Replace(",", "_COMMA")
```

Comment:
Missing an underscore in `_COMMA_`.

Reply:
I did this intentionally, as you always have a space after a comma, you'd always end up with `_COMMA__`, and technically could end up with a triple underscore if you did something weird. But I'll probably change the first argument to `", "` to avoid that instead.